src/core/telemetry/decorators.ts

```
@@ -0,0 +1,297 @@
+                    finishReason: finishData.finishReason,
+                    toolCalls: finishData.toolCalls,
+                    toolResults: finishData.toolResults,
+                    warnings: finishData.warnings,
+                    ...(finishData.object !== undefined && { object: finishData.object }),
+                };
+
+                span.setAttribute(`${spanName}.result`, JSON.stringify(telemetryData));
+                span.setStatus({ code: SpanStatusCode.OK });
+                span.end();
+            } catch (error) {
+                debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)